docs/content/charts/configs/Legend.js

```diff
@@ -21,5 +21,12 @@ export default {
         description: <cx><Md>
             Switch to vertical mode.
         </Md></cx>
+    },
+
+    shape: {
+        type: 'string',
+        description: <cx><Md>
+            Specifies the shape of the legend. By default, the shape of the series in the chart is used for the legend. `circle`, `square`, `triangle`, etc.
```

Comment:
The shape of the legend doesn't sound right. Perhaps, the shape of the symbol in the legend entries.